ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -283,6 +289,13 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
     => Set (SL.Credential 'SL.Staking (EraCrypto era))
     -> BlockQuery (ShelleyBlock proto era) (VoteDelegatees (EraCrypto era))
 
+  -- | Obtain a snapshot of big ledger peers. CLI can serialize these,
+  -- and if made available to the node by topology configuration,
+  -- the diffusion layer can use these peers when syncing up from scratch
+  -- or stale ledger state - especially useful for Genesis mode
+  GetPeerSnapshot
```

Comment:
Perhaps `GetBigLedgerPeerSnapshot` or something that is similarly more specific.